.github/workflows/UpdateReadme.yml

```diff
@@ -0,0 +1,90 @@
+---
+name: Update README
+
+on:
+  push:
+    paths:
+      - 'participants/**/*.yml'
+  schedule:
+    - cron: '0 0 1 1 *'
```

Comment:
This runs once every year. Is that intentional?

Reply:
Yes, it is. See https://github.com/benbarth/hacktoberfest-swag/pull/118#issuecomment-701383741